### DISCUSSION TITLE
How do I convert sound data from bytes to a file/object?

### DISCUSSION THREAD

**Discussion by glgrig** [streamlink/streamlink#3613] [Q&A]
Hello, I am trying to work with streamlink to record audio from twitch. I want to convert the data I am getting into a wav or mp3 object in python, is there a way to do that with the streamlink library?

**Comment by gravyboat** [comment]
Outside of suggesting the `audio_only` quality option this is outside the scope of our support and audio conversion is not directly supported by Streamlink. You'll have to handle the conversion yourself.

**Reply by glgrig** [reply]
Thanks fot the answer, I will try to solve it with ffmpeg.
